### ISSUE TITLE
Qickstart code does not work

### ISSUE THREAD

**Issue by BryanHunt** [Addepar/ember-table#650]
If you follow the quickstart documentation (https://opensource.addepar.com/ember-table/latest/docs/quickstart), the code fails in many ways.

Steps to reproduce:
```
ember new table-test
cd table-test
ember g component demo-component

Copy the js code from the quickstart documentation and paste it into the demo-component.js file
Copy the hbs code from the quikstart documentation and past it into the demo-component.hbs file

ember s
```
The result:

```
Build Error (broccoli-persistent-filter:Babel > [Babel: table-test]) in table-test/components/demo-component.js

/Users/bhunt/Development/Nvidia/ui/table-test/table-test/components/demo-component.js: Support for the experimental syntax 'classProperties' isn't currently enabled (4:11):

  2 |
  3 | class DemoComponent extends Component {
> 4 |   columns = [
    |           ^
  5 |     {
  6 |       name: `First Name`,
  7 |       valuePath: `firstName`

Add @babel/plugin-proposal-class-properties (https://git.io/vb4SL) to the 'plugins' section of your Babel config to enable transformation.
```

If you click on the link https://git.io/vb4SL, the instructions there say to:

```
npm install --save-dev @babel/plugin-proposal-class-properties
```

After getting some help from the Ember folks on Discord, I was advised to:
```
ember install ember-decorators
```

That fixes the build error.  Running then gives the following runtime error:
```
Expected to find: 'component:demo-component' within 'table-test/components/demo-component' but got 'undefined'. Did you forget to 'export default' within 'table-test/components/demo-component'?
```

Changing your example code to:
```
export default class DemoComponent
```
fixes that runtime error.  Running again then gives the following runtime error:
```
Property set failed: object in path "unwrappedApi" could not be found.
```

**Comment by cyk** [comment]
@BryanHunt, I believe this is fixed in 2.0.0-beta.7. Can you please try these steps in this version?

**Comment by bantic** [comment]
I'm going to close this for now, since there hasn't been an update for a little while, but please feel free to re-open if needed!
